packages/ui-voip/src/components/VoipPopup/views/VoipDialerView.tsx

```diff
@@ -44,6 +45,8 @@ const VoipDialerView = ({ position }: VoipDialerViewProps) => {
 			</Footer>
 		</Container>
 	);
-};
+});
+
+VoipDialerView.displayName = 'VoipDialerView';
```

Comment:
Using a named function as the render function in `forwardRef()` does the same trick.